lib/fog/storage/google_json/real.rb

```diff
@@ -18,11 +18,11 @@ def initialize(options = {})
         end
 
         def signature(params)
-          string_to_sign = <<-DATA
-#{params[:method]}
-#{params[:headers]['Content-MD5']}
-#{params[:headers]['Content-Type']}
-#{params[:headers]['Date']}
+          string_to_sign = <<-DATA.unindent
```

Comment:
This seems ok, but we need to make sure this doesn't break the signature, as has happened multiple times before. I did a quick sanity check, but just to check - did you test this in a real scenario?